.changeset/tiny-spoons-itch.md

```diff
@@ -0,0 +1,5 @@
+---
+'myst-cli': patch
```

Comment:
As far as I can tell, `changeset` only plays nicely with JS packages. I tried to add `mystmd-py` as a changeset package and it errored.